docs/keys-service.md

```diff
@@ -84,9 +84,31 @@ Required. See [API authentication](#api-authentication).
 
 ---
 
+### Delete Symmetric Key
+
+`DELETE /key?api-version=2021-05-01`
+
+#### Authentication
+
+Not required.
```

Comment:
Should these delete APIs require authentication? AFAIK all other key R/W operations do.

Reply:
They don't require authentication because they need a key handle. Same as encrypt, etc.

Reply:
But is allowing anyone with the key handle to delete the key safe? I suppose we do trust whoever has the key handles since GET requires authentication.